class-parsely-recommended-widget.php

```diff
@@ -9,24 +34,34 @@ public function __construct() {
 		parent::__construct( 'Parsely_Recommended_Widget', 'Parsely Recommended Widget', $widget_options );
 	}
 
+	/**
+	 * This is the widget function
+	 *
+	 * @category   Function
+	 * @package    WordPress
+	 * @subpackage Parse.ly
+	 * @param array $args Widget Arguments.
+	 * @param array $instance Values saved to the db.
+	 */
 	public function widget( $args, $instance ) {
 		$title = apply_filters( 'widget_title', $instance['title'] );
 
 		$instance['display_options'] = ! empty( $instance['display_options'] ) ? $instance['display_options'] : array();
-		echo $args['before_widget'] . $args['before_title'] . $title . $args['after_title'];
+		$allowed_tags                = wp_kses_allowed_html( 'post' );
+		$title_html                  = $args['before_widget'] . $args['before_title'] . $title . $args['after_title'];
+		echo wp_kses( $title_html, $allowed_tags );
```

Comment:
For some reason this seems to be stripping the thumbnail images from the widget. But when I log `$allowed_tags`, it includes image elements and src attributes (which is the only attribute we use. So I haven't figured out yet why they're getting stripped.

Reply:
Weird, I was able to see the thumbnail images in the widget. I'll look at that.

Reply:
hmm - maybe it's something in my environment. Maybe we can look at it together.

Reply:
confirmed it was css-related handling of the no thumbnail setting